web/models/package_metadata.ex

```diff
@@ -6,10 +6,11 @@ defmodule HexWeb.PackageMetadata do
     field :licenses, {:array, :string}
     field :links, :map
     field :maintainers, {:array, :string}
+    field :extra, :map
```

Comment:
btw, when we update to latest Ecto, this map could be typed. It goes back to @mobileoverlord's question if this should be map of maps, or map of arrays (ideally not both, so that we can get type safety)